docs/source/index.rst

```diff
@@ -128,3 +128,4 @@ Some quick links:
    additional/release.rst
    additional/faq.rst
    additional/glossary.rst
+   additional/privacy-strategy.rst
```

Comment:
I think I would personally rather put it under the "What is Substra" section, rather than hidden at the bottom. But happy to be challenged.

Reply:
I think this is quite specific and not key to what substra is - not everyone who uses substra would be keen to read this